itests/kit/ensemble_presets.go

```diff
@@ -77,6 +80,33 @@ func EnsembleOneTwo(t *testing.T, opts ...interface{}) (*TestFullNode, *TestMine
 	return &full, &one, &two, ens
 }
 
+// EnsembleOneTwo creates and starts an Ensemble with two full nodes and two miners participating in F3.
+// It does not interconnect nodes nor does it begin mining.
+//
+// This ensemble only configured miners to participate in F3, and it expects to provide,
+// F3Enabled as an option to configure the parameters for F3. If F3Enabled is nos passed as a node option
+// to the ensemble, building the miner fails.
+func EnsembleF3(t *testing.T, opts ...interface{}) (*TestFullNode, *TestMiner, *TestMiner, *Ensemble) {
+	opts = append(opts, WithAllSubsystems())
+
+	eopts, nopts := siftOptions(t, opts)
+
+	var (
+		full     TestFullNode
+		one, two TestMiner
+	)
+	minerOpts := append(nopts, ConstructorOpts(
+		node.Override(node.F3Participation, modules.F3Participation),
```

Comment:
could you just add a new option to enable f3 participation to the ensemble rather than an entire new constructor just to add this?

Reply:
oh, you have one below, `F3Enabled`, so why is this needed?